Livecheckables/gtk+.rb

```diff
@@ -0,0 +1,4 @@
+class Gtkx
+  livecheck :url   => "https://download.gnome.org/sources/gtk+/",
+            :regex => /gtk\+-(2\.24(?:\.\d+)+)\.t/
```

Comment:
```suggestion
            :regex => /gtk\+-(2\.([0-8]\d*?)?[02468](?:\.\d+)*?)\.t/
```

We only need to restrict the major version (to 2) and we don't need to restrict the minor version to a specific version (e.g., 24).

Rather,`gtk+` uses the GNOME version scheme, where stable versions are indicated by an even-numbered minor version below 90, so we want to enforce that like we do in the `download.gnome.org` strategy's regex. The only difference in the suggested regex here and the `download.gnome.org` strategy's regex is the hardcoded major version of 2.